src/libsyntax/parse/parser.rs

```diff
@@ -793,6 +793,9 @@ impl<'a> Parser<'a> {
                         return Err(err);
                     }
                 }
+                if is_raw {
+                    self.sess.raw_identifier_spans.borrow_mut().push(self.span);
```

Comment:
I'm not entirely sure all identifiers go through `parse_ident_common`, it's probably better to do in the lexer when an identifier is created.

Reply:
Oh huh. I didn't catch that `ParseSess` was available in the lexer too. Definitely a better place for it.